MIGRATION.md

```diff
@@ -0,0 +1,239 @@
+
+**V1 Initialization**
+```swift
+import Datadog
+
+Datadog.initialize(
+    appContext: .init(),
+    trackingConsent: .granted,
+    configuration: Datadog.Configuration
+        .builderUsing(
+            clientToken: "<client token>",
+            environment: "<environment>"
+        )
+        .set(serviceName: "<service name>")
+        .build()
+```
+**V2 Initialization**
+```swift
+import DatadogCore
+
+Datadog.initialize(
+    with: Datadog.Configuration(
+        clientToken: "<client token>",
+        env: "<environment>",
+        service: "<service name>"
+    ), 
+    trackingConsent: .granted
+)
+```
+
+API changes:
+
+|`1.x`|`2.0`|
+|---|---|
+|`Datadog.Configuration.Builder.set(serviceName:)`|`Datadog.Configuration.service`|
+|`Datadog.Configuration.Builder.set(batchSize:)`|`Datadog.Configuration.batchSize`|
+|`Datadog.Configuration.Builder.set(uploadFrequency:)`|`Datadog.Configuration.uploadFrequency`|
+|`Datadog.Configuration.Builder.set(proxyConfiguration:)`|`Datadog.Configuration.proxyConfiguration`|
+|`Datadog.Configuration.Builder.set(encryption:)`|`Datadog.Configuration.encryption`|
+|`Datadog.Configuration.Builder.set(serverDateProvider:)`|`Datadog.Configuration.serverDateProvider`|
+|`Datadog.AppContext(mainBundle:)`|`Datadog.Configuration.bundle`|
+
+## Logs Product Changes
+
+All the classes related to Logs are now strictly in the `DatadogLogs` module. You will first need to enable the product:
+
+```swift
+import DatadogLogs
+
+Logs.enable()
+```
+
+Then, you can create a logger instance:
+
+```swift
+import DatadogLogs
+
+let logger = Logger.create(with: Logger.Configuration(name: "<logger name>"))
+```
+
+API changes:
+
+|`1.x`|`2.0`|
+|---|---|
+|`Datadog.Configuration.Builder.setLogEventMapper`|`Logs.Configuration.eventMapper`|
+|`Datadog.Configuration.Builder.set(loggingSamplingRate:)`|`Logs.Configuration.eventMapper`|
+|`Logger.Builder.set(serviceName:)`|`Logger.Configuration.service`|
+|`Logger.Builder.set(loggerName:)`|`Logger.Configuration.name`|
+|`Logger.Builder.sendNetworkInfo`|`Logger.Configuration.networkInfoEnabled`|
+|`Logger.Builder.bundleWithRUM`|`Logger.Configuration.bundleWithRumEnabled`|
+|`Logger.Builder.bundleWithTrace`|`Logger.Configuration.bundleWithTraceEnabled`|
+|`Logger.Builder.sendLogsToDatadog = false`|`Logger.Configuration.remoteSampleRate = 0`|
+|`Logger.Builder.set(datadogReportingThreshold:)`|`Logger.Configuration.remoteLogThreshold`|
+|`Logger.Builder.printLogsToConsole(:, usingFormat)`|`Logger.Configuration.consoleLogFormat`|
+
+## APM Trace Product Changes
+
+All the classes related to Trace are now strictly in the `DatadogTrace` module. You will first need to enable the product:
+
+```swift
+import DatadogTrace
+
+Trace.enable()
+```
+
+Then, you can access the shared Tracer instance:
+
+```swift
+import DatadogTrace
+
+let tracer = Tracer.shared()
+```
+
+API changes:
+
+|`1.x`|`2.0`|
```

Comment:
nice idea of having changes as table

Reply:
I've also move the `Support for multiple SDK instances` section to the end of the document to not confuse users.

Reply:
@0xnm you could update the Android guide to make the experience consistent.

Reply:
done.